### ISSUE TITLE
When logging in to GNOME, getting an error about rekeying KBFS

### ISSUE THREAD

**Issue by lokedhs** [keybase/keybase-issues#3162]
The error seems almost identical to https://github.com/keybase/client/issues/3447

The message appears every time I log in, and if I enter the paper key I get a message saying "Success" and a statement to the effect that the folders are being rekeyed. Nothing happens when I click the "OK" button and I can't close the window without killing the process.

Version reported by `keybase version`:

```
Client:  1.0.39-20180117172434+35983dc56
Service: 1.0.39-20180117172434+35983dc56
```

Logs are uploaded as log id: `ade61ca6f4d89d3199c8151c`

**Comment by maxtaco** [comment]
cc: @strib there might be a broken folder in there
cc: @chrisnojima the UI might be broken here

**Comment by strib** [comment]
At the time, the user was accessing the folder from a new device that hadn't been keyed for a previous device yet.  The provisioning device was `em-desktop`, and I don't know why it didn't rekey the folder right away -- @lokedhs could you please do a `keybase log send` from `em-desktop` and I'll try to figure out what happened.

However, at this point it looks like the folder eventually got successfully rekeyed this morning (about 2.5 hours ago) by `em-desktop` -- maybe it was turned off soon after the provision and just got turned on this morning?

**Comment by lokedhs** [comment]
You are right. KBFS was not running on em-desktop. It seemed to start once I ran the GUI application (which is something I never normally do, is there a way to do this from the commandline?).

So the question is why the dialog box on em-dell was hanging.
